lms/static/js/spec/edxnotes/views/shim_spec.js

```diff
@@ -29,15 +29,15 @@ define([
             loadFixtures('js/fixtures/edxnotes/edxnotes_wrapper.html');
             highlights = [];
             annotators = [
-                NotesFactory.factory($('div#edx-notes-wrapper-123').get(0), {
+                NotesFactory.factory($('#edx-notes-wrapper-123').get(0), {
                     endpoint: 'http://example.com/'
                 }),
-                NotesFactory.factory($('div#edx-notes-wrapper-456').get(0), {
+                NotesFactory.factory($('#edx-notes-wrapper-456').get(0), {
                     endpoint: 'http://example.com/'
                 })
             ];
             _.each(annotators, function(annotator, index) {
```

Comment:
@polesye The `index` argument is not used anymore, please remove it:
`
_.each(annotators, function(annotator) {
…
`